packages/nc-gui/assets/style.scss

```diff
@@ -32,6 +32,10 @@
   height: var(--topbar-height) !important;
 }
 
+.anticon-check-circle {
+  @apply !relative top-[-1px] left-0;  
+}
```

Comment:
I didn't see this class in use. Is it overriding a system class? Please share some steps to tell how to verify it.

Reply:
It is used to align the check-mark icon in ant-message

Reply:
ok. please add the comment in the code then.